pkg/fakerp/customer_handlers.go

```diff
@@ -95,10 +96,10 @@ func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
 		// long-running operation can exit successfully.
 		if resp.StatusCode == http.StatusOK {
 			s.log.Infof("deleted resource group %s", resourceGroup)
-			s.write(nil)
+			s.store.Delete()
```

Comment:
I'd like to get rid of the asynchronous delete - it adds complexity for no real benefit.  That does not imply removing NO_WAIT, which I'd like to keep.

Reply:
done